recipes/rapid-pe/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "rapid-pe" %}
+{% set version = "0.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rapid_pe-{{ version }}.tar.gz
+  sha256: 53739bd328779f8484cfc1887ba18aa7b8a0b0ea82b064bb4866b2cbc6eaa518
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - h5py
+    - lalsuite
+    - lscsoft-glue
+    - matplotlib-base
+    - numpy
+    - python
+    - scikit-learn
+    - scipy
+    - six
+
+test:
+  imports:
+    - rapid_pe
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://git.ligo.org/rapidpe-rift/rapidpe/
+  summary: 'RapidPE: The original low-latency gravitational wave parameter estimation code.'
+  license: GPL-2.0
```

@dwysocki, this is ambiguous, can you change this to one of

```suggestion
  license: GPL-2.0-only
```

or

```suggestion
  license: GPL-2.0-or-later
```

---

Thanks for catching this, I've taken your latter suggestion in the latest commit